agenthub/codeact_agent/action_parser.py

```diff
@@ -179,4 +180,13 @@ def check_condition(self, action_str: str) -> bool:
         return True
 
     def parse(self, action_str: str) -> Action:
-        return MessageAction(content=action_str, wait_for_response=True)
+        is_autonomous = config.agent.is_autonomous
+        if is_autonomous:
+            user_msg = (
+                '\n----------\n'
+                'Please continue working on the task on whatever approach you think is suitable.\n'
+                'If you think you have solved the task, you can give <finish> to end the interaction.\n'
+                'IMPORTANT: YOU SHOULD NEVER ASK FOR HUMAN HELP.\n'
+            )
+            action_str += user_msg
```

Comment:
this is too hacky. This class parses the response from LLM, and we shouldn't append response here.